dan_layer/integration_tests/src/test_consensus.rs

```diff
@@ -644,6 +644,34 @@ async fn test_leader_fails_only_foreignly() {
     instance1.assert_shuts_down_safely().await;
 }
 
+#[tokio::test(flavor = "multi_thread", worker_threads = 2)]
```

Comment:
```suggestion
#[tokio::test(flavor = "multi_thread", worker_threads = 2)]
#[should_panic]
```

Reply:
Done. Reworked the test little bit to panic.